src/main/java/org/spongepowered/api/service/economy/Account.java

```diff
@@ -0,0 +1,231 @@
+    public double getBalance();
+
+    /**
+     * Sets the balance stored by this {@link Account}
+     * as the given value.
+     * 
+     * <p>The amount set cannot exceed two decimal places
+     * (e.g. <pre>129.54</pre>); as such, the double passed
+     * as the amount will be rounded up or down if it
+     * violates the former.</p>
+     * 
+     * <p>The amount is allowed to be negative.</p>
+     * 
+     * @param amount a double value
+     */
+    public void setBalance(double amount);
+
+    /**
+     * Pays the target {@link Account} the given amount
+     * of currency by taking the amount from this
+     * Account.
+     * 
+     * <p>If this Account results in holding a negative
+     * balance due to this operation, this will return
+     * true. This does not mean the operation is
+     * cancelled.</p>
+     * 
+     * <p>A negative amount parameter will result in an
+     * IllegalArgumentException.</p>
+     * 
+     * @param target the Account to send the amount of
+     *        money to
+     * @param amount a double representative of the amount
+     *        to give to the target Account
+     * 
+     * @return true if this Account's balance turned
+     *         negative
+     * 
+     * @throws IllegalArgumentException if the amount to pay
+     *         is negative
+     */
+    public boolean pay(Account target, double amount);
+
+    /**
+     * Removes the given sum of currency from this
+     * {@link Account}.
+     * 
+     * <p>If this Account results in holding a negative
+     * balance due to this operation, this will return
+     * true. This does not mean the operation is
+     * cancelled.</p>
+     * 
+     * <p>A negative amount parameter will result in an
+     * IllegalArgumentException.</p>
+     * 
+     * @param amount a double representing the amount to
+     *        remove from the account
+     * 
+     * @return true if this Account's balance turned
+     *         negative
+     * 
+     * @throws IllegalArgumentException if the amount to
+     *         remove is negative
+     */
+    public boolean elicit(double amount);
+
+    /**
+     * Removes the given sum of currency from this
+     * {@link Account} and sends it to the destination
+     * Account.
+     * 
+     * <p>If this Account results in holding a negative
+     * balance due to this operation, this will return
+     * true. This does not mean the operation is
+     * cancelled.</p>
+     * 
+     * <p>A negative amount parameter will result in an
+     * IllegalArgumentException.</p>
+     * 
+     * @param destination the Account to send the
+     *        withdrawn currency to
+     * @param amount a double representing the amount to
+     *        withdraw
+     * 
+     * @return true if this Account's balance turned
+     *         negative
+     * 
+     * @throws IllegalArgumentException if the amount to
+     *         remove is negative
+     */
+    public boolean withdraw(Account destination, double amount);
```

Comment:
What is the difference between pay and withdraw?
From the javadocs i only get 'transfer the given amount of money to the target account'.